src/getting_started/io.md

```diff
@@ -1,22 +1,55 @@
-# I/O: Runner file handling
+# I/O: Runner File Handling
 
-TODO:
+When you're using Styx wrapper functions with the default runners, you won't have to worry too much about I/O since a lot of it gets handled automatically for you.
 
-- Explain output tuple, root attribute.
+## Basic File Access
 
-## _Default_ runner behaviour
+Take this example:
 
-While custom Styx runners allow users to implement file I/O handling any way they want, the 'default' runners (`LocalRunner` as well as `DockerRunner` and `SingularityRunner`) work very similarly by default.
+```python
+outputs = fsl.bet(infile="brain.nii", fractional_intensity=0.5)
 
-They create a directory (default path `styx_temp/` in the current working directory) in which a folder gets created for every Styx function call. These will look like this:
+outputs.mask_file  # File path to an individual output
+```
+
+The `outputs` object gives you structured access to generated files through properties that support autocompletion and include helpful documentation.
+
+## Dynamic File Access
+
+Sometimes though, either when a descriptor hasn't been fully implemented or if the output structure is more complex, there may not be an output property available. For these cases, you can use the `outputs.root` property, which always points to the output directory:
```

Comment:
Would rephrase the first sentence slightly...maybe something like

"Sometimes, when a descriptor hasn't been fully implemented or if output structure is more complex, an output property may not be available.

Reply:
```suggestion
Sometimes, when a descriptor hasn't been fully implemented or if the output structure is more complex, an output property may not be available.. For these cases, you can use the `outputs.root` property, which always points to the output directory:
```